server/services/licence/licenceStatusTypes.ts

```diff
@@ -0,0 +1,92 @@
+import { taskState } from '../config/taskState'
+import { licenceStage } from '../config/licenceStage'
+
+export type LicenceStatus = {
+  stage: licenceStage
+  postApproval: boolean
+  decisions: Decisions
+  tasks: Tasks
+}
+
+export type Decisions = {
+  // eligibility
+  eligible?: boolean
+  excluded?: boolean
+  unsuitable?: boolean
+  unsuitableResult?: boolean
+  exceptionalCircumstances?: boolean
+  insufficientTime?: boolean
+  insufficientTimeContinue?: boolean
+  insufficientTimeStop?: boolean
+
+  // BASS address related
+  bassReferralNeeded?: boolean
+  bassAreaSpecified?: boolean
+  bassAreaSuitable?: boolean
+  bassAreaNotSuitable?: boolean
+  bassAccepted?: 'Yes' | 'Unavailable' | 'Unsuitable'
```

Comment:
`bassAccepted` is copied straight over from the licence for... reasons?

Reply:
Perhaps this should be a named type like YesNo.